webextension/background.js

```diff
@@ -342,9 +350,67 @@ async function shouldPromptUser(tabId, url) {
   }
 }
 
-function backgroundSendReport(report) {
-  console.info("Would submit this report: ", report);
-  return Promise.resolve();
+function yesOrNo(bool) {
+  return bool ? "yes" : "no";
+}
+
+function pingTelemetry(message) {
+  if (Config.testingMode) {
+    console.info("Pinging telemetry: ", message);
+    return false;
+  }
+
+  // TBD
+  return true;
+}
+
+function backgroundSendReport(data) {
+  data.type = browser.i18n.getMessage(`issueLabel${data.type}`);
+
+  const body = ["url", "type", "appVersion", "channel", "platform", "buildID",
+                "experimentBranch", "description"].map(function(name) {
+      const label = browser.i18n.getMessage(`detailLabel_${name}`);
+      const value = data[name] || "";
+      return `${label} ${value}`;
```

Comment:
Ah, oops. Can we add asterisks, so the labels will be bolded when seen on GitHub?

Sorry, I didn't request this initially.

```
return `**${label}**: ${value}`;
```